samples/AnomalyDetector/src/Generated/AnomalyDetectorModelFactory.cs

```diff
@@ -233,6 +233,15 @@ public static MultivariateBatchDetectionResultSummary MultivariateBatchDetection
             return new MultivariateBatchDetectionResultSummary(status, errors?.ToList(), variableStates?.ToList(), setupInfo, serializedAdditionalRawData: null);
         }
 
+        /// <summary> Initializes a new instance of <see cref="Models.ErrorResponse"/>. </summary>
+        /// <param name="code"> The error code. </param>
+        /// <param name="message"> The message explaining the error reported by the service. </param>
+        /// <returns> A new <see cref="Models.ErrorResponse"/> instance for mocking. </returns>
+        public static ErrorResponse ErrorResponse(string code = null, string message = null)
```

Comment:
this shows up because of our rules on "who would have an entry in model factory"
the changes on `ErrorResponse` is that its usage changed from roundtrip to output, therefore the accessibility of the class did not change, but only its public ctor changed to internal and all its properties no longer have a setter now.

The rules of model factory is that when a model has a property without setter, it will show up in the model factory.
Previously every property in ResponseError is settable therefore it was not there, now it changed to output and all its properties do not have a setter now, therefore it shows up